spec/models/video_spec.rb

```diff
@@ -2,24 +2,27 @@
 
 describe Video do
   it {should belong_to (:category)}
+  it {should have_many(:reviews).order("created_at DESC")}
   it {should validate_presence_of (:title)}
   it {should validate_presence_of (:description)}
 
   describe "#search_by_title" do
-    context "when there's no match" do
+    let(:subject) {Video.search_by_title("some title")}
+
+    context "no match" do
       it "returns an empty array" do
-        expect(Video.search_by_title("some title")).to eq([])
+        expect(subject).to eq([])
```

Comment:
I'm sorry. I'm not sure I understand your comment the way you meant it. 
The way I understand it is that I should use subject but not let. Is that right ?